src/index.tsx

```diff
@@ -22,6 +22,7 @@ export type Error = {
   type: ErrorType
   code: string
   message: string
+  pendingRetry?: boolean
```

Comment:
Doesn't make a practical difference but for the sake of consistency should not be optional - Link sends it every time and it's a mandatory field on the other SDKs as well so devs know they can rely on it being there.

Reply:
ok!